tests/config_test.yml

```diff
@@ -6,7 +6,7 @@ framework:
         utf8: true
     secret: 'fake_secret'
     http_method_override: true
-    handle_all_throwables: true
+    handle_all_throwables: false
```

Comment:
The option "handle_all_throwables" is not defined in previous versions. You will receive an error: "Unrecognized option 'handle_all_throwables' under 'framework'."